### PR TITLE
Added React Native project setup 

### PR DESCRIPTION
**Pull request by maryjabro**
Added the initial setup for the React Native project within the VitalTrackApp folder including the basic structure for Android platform. Needs reviewal to ensure correctness

### CODE REVIEW

File: VitalTrackApp/App.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  Header,
+  LearnMoreLinks,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
+
+type SectionProps = PropsWithChildren<{
+  title: string;
+}>;
+
+function Section({children, title}: SectionProps): React.JSX.Element {
```
Comment (MerwinAamir1) [inline VitalTrackApp/App.tsx:32]: Can we remove all this boilerplate

**Comment by MerwinAamir1** [comment]
Have we tested this runs?

**Comment by MerwinAamir1** [comment]
Can we get rid of all images, css, etc.. (basically lets start with a blank canvas)

**Comment by maryjabro** [comment]
> Can we get rid of all images, css, etc.. (basically lets start with a blank canvas)

We can do that, however it starts off with a template that teaches you how to navigate. Im thinking it might be helpful for whoever is starting on the front-end to see.  See images of the emulator below: 
Screenshot 2024-09-15 at 2 18 58 PMScreenshot 2024-09-15 at 2 19 56 PM

**Comment by MerwinAamir1** [comment]
approved.

**Pull request merged**